source/user-guide/portable-mode.rst

```diff
@@ -0,0 +1,42 @@
+Using portable mode
+===================
+
+Sometimes it can be useful for data of an application to travel along with the application, for example to put the application on a USB stick that can be used with different computers. In the windows world, this concept is known as “portable applications”.
+
+Normally the application contained inside an AppImage will store its configuration files wherever it always stores them (most frequently somewhere inside :code:`$HOME`). In other words, the fact that an application is contained inside an AppImage normally does not change where the application stores its data.
+
+However, there is functionality in newer AppImages that can make the application's data travel along with the application, if certain directories are present *next to the AppImage file*.
+
+If you invoke an AppImage built with a recent version of AppImageKit and have one of these special directories in place, then the configuration files will be stored alongside the AppImage. This can be useful for portable use cases, e.g., carrying an AppImage on a USB stick, along with its data.
+
+- If there is a directory with the same name as the AppImage plus :code:`.home`, then :code:`$HOME` will automatically be set to it before executing the payload application
+- If there is a directory with the same name as the AppImage plus :code:`.config`, then :code:`$XDG_CONFIG_HOME` will automatically be set to it before executing the payload application
+
+
+.. _example:
```

Comment:
This is quite a _bad_ reference title, it's too generic. I doubt anyone will ever need to reference that section, so it's probably best to remove it.